admin/src/auth.js

```diff
@@ -111,8 +111,33 @@ class Auth {
             .catch(() => {
                 showError("<h2>Inloggningen misslyckades</h2>Kunde inte kommunicera med servern.");
             });
+
+    }
+
+    update_member_info(tag) {
+        fetch(config.apiBasePath + "/member/send_updated_member_info",
+            {
+                method: "POST",
+                body: JSON.stringify({ member_number: tag }),
+                headers: { 'Content-Type': 'application/json; charset=UTF-8' },
```

Comment:
Are you sure we don't have a helper for this somewhere?

Reply:
I can't find any helper function for this. These lines are written in the same way as to other similar functions in #auth.js